### ISSUE TITLE
Custom panels - 3-position knob not working properly

### ISSUE THREAD

**Issue by DrprofLuigi** [DonBruce64/MinecraftTransportSimulator#1840]
![2024-08-24_17 25 06](https://github.com/user-attachments/assets/f9e88a48-0977-4e04-ac23-789f869669ae)

![2024-08-24_17 27 15](https://github.com/user-attachments/assets/5a71f49b-edaf-4ed7-8403-bffbfe4837f9)

![unuparts_panel_unu_truck_heavy](https://github.com/user-attachments/assets/c87f62e7-33d7-434d-b6d1-660e6dae578b)

I have the wiper knob in the above panel tied to `rain_strength`, however it only moves one notch. Laura also reported the increment function not working properly, though it could simply be an animation issue.

**Comment by DonBruce64** [comment]
So panels only work with 0/1 states.  Boolean, not variable.  The only variable-state stuff that's there is hard-coded since it's stuff that's multi-variable, like engine magneto/starter and the like.

TLDR, functioning as expected.  Throw a feature thread in the Discord if you want this.
